praktikumsplaner-frontend/frontend/src/components/TheSnackbar.vue

```diff
@@ -2,18 +2,26 @@
     <v-snackbar
         id="snackbar"
         v-model="show"
-        :color="color"
+        :color="backgroundColor"
         :timeout="timeout"
     >
-        {{ message }}
-        <v-btn
-            v-if="color === 'error'"
-            color="primary"
-            text
-            @click="show = false"
-        >
-            Schließen
-        </v-btn>
+        <v-row class="snackbarContent">
+            <v-col class="message"> {{ message }}</v-col>
+            <v-col>
+                <v-btn
+                    v-if="
+                        backgroundColor === 'error' ||
+                        backgroundColor === 'warning' ||
+                        backgroundColor === 'info'
```

Comment:
why is the condition of the button connected to the color and not to the level?

Maybe create a function like `nonSuccessMessage` to combine the checks